tests/testparse.cpp

```diff
@@ -87,15 +88,16 @@ void TestParse::testParseCSV()
 void TestParse::testParseDivingLog()
 {
 	// Parsing of DivingLog import from SQLite database
-	sqlite3 *handle;
+	sqlite3 *handle = NULL;
+	ScopedShutdownHelper scoped_shutdown( [&](){
+						sqlite3_close(handle);
+					});
```

Comment:
This is one of those "features" in C++ that I dislike. I understand why people love it, I don't.
I think it makes much more sense to re-design the tests so that they aren't interdependent,